pulp_container/app/tasks/sync_stages.py

```diff
@@ -515,6 +516,25 @@ async def create_listed_manifest(self, manifest_data):
         )
         return {"manifest_dc": man_dc, "platform": platform, "content_data": content_data}
 
+    async def create_config_blob(self, blob_data):
+        digest = blob_data.get("digest") or blob_data.get("blobSum")
```

Comment:
I could not find the `blobSum` field in https://github.com/opencontainers/image-spec/blob/main/config.md. Can you verify this exists?